lib/reactotron-react-js/package.json

```
@@ -0,0 +1,126 @@
+{
+  "name": "reactotron-react-js",
+  "version": "2.1.1",
+  "description": "A development tool to explore, inspect, and diagnose your React JS/DOM apps.",
+  "author": "Infinite Red",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/infinitered/reactotron/issues"
+  },
+  "homepage": "https://github.com/infinitered/reactotron-react-js",
+  "repository": "https://github.com/infinitered/reactotron-react-js",
+  "files": [
+    "dist",
+    "LICENSE",
+    "README.md",
+    "reactotron-react-js.d.ts"
+  ],
+  "main": "dist/index.js",
+  "types": "./dist/types/index.d.ts",
+  "scripts": {
+    "test": "jest",
+    "test:watch": "jest --watch --notify",
+    "format": "prettier --write {**,.}/*.ts",
+    "build": "npm-run-all clean tsc compile",
+    "build:dev": "npm-run-all clean tsc compile:dev",
+    "clean": "trash dist",
+    "lint": "eslint src --ext .ts,.tsx",
+    "compile": "NODE_ENV=production rollup -c",
+    "compile:dev": "NODE_ENV=development rollup -c",
+    "tsc": "tsc",
+    "ci:lint": "yarn lint",
+    "ci:test": "yarn test",
```

We need `--runInBand` to run the tests not in parallel or else running the tests in all the workspaces with `yarn tests` gets wonky.

```suggestion
    "ci:test": "yarn test --runInBand",
```